src/main/java/redis/clients/jedis/commands/AdvancedBinaryJedisCommands.java

```diff
@@ -64,5 +64,11 @@
 
   List<byte[]> aclCat(byte[] category);
 
+  List<byte[]> aclLogBinary();
+
+  List<byte[]> aclLogBinary(int limit);
```

Comment:
Same as line 67.

Reply:
Remove the binary methods that are not necessary now.